<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntelligentScissor</class>
 <widget class="QMainWindow" name="IntelligentScissor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IntelligentScissor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/image/scissor (1).ico</normaloff>:/icon/image/scissor (1).ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1311</width>
         <height>537</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>1311</width>
         <height>529</height>
        </size>
       </property>
       <widget class="my_label" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>991</width>
          <height>471</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionScissorStart"/>
   <addaction name="actionScissorStop"/>
   <addaction name="actionScissorUndoLast"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveContour"/>
    <addaction name="actionSaveMask"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/image/new.ico</normaloff>:/icon/image/new.ico</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icon/image/open_file.ico</normaloff>:/icon/image/open_file.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveContour">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/image/save_mask.ico</normaloff>:/icon/image/save_mask.ico</iconset>
   </property>
   <property name="text">
    <string>Save Contour</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSaveMask">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/image/save_counter.ico</normaloff>:/icon/image/save_counter.ico</iconset>
   </property>
   <property name="text">
    <string>Save Mask</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/image/exit.ico</normaloff>:/icon/image/exit.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionOriginal_Image">
   <property name="text">
    <string>Original Image</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="iconText">
    <string notr="true">放大</string>
   </property>
   <property name="toolTip">
    <string notr="true">放大</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string notr="true">缩小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionScissorStart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
   <property name="toolTip">
    <string notr="true">开始</string>
   </property>
  </action>
  <action name="actionScissorStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
   <property name="toolTip">
    <string notr="true">完成</string>
   </property>
  </action>
  <action name="actionScissorUndoLast">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string notr="true">撤销</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionHow_To">
   <property name="text">
    <string>Usage</string>
   </property>
   <property name="toolTip">
    <string notr="true">Usage</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_label</class>
   <extends>QLabel</extends>
   <header location="global">my_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
